CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 - Add support for unindexed (and uncompressed) VCFs (#337)
 
+- Remove explicit sample, contig and filter lists from the schema (#343)
```

Comment:
This is a breaking schema format change, so will require existing ICFs to be recreated. I listed these as "Breaking changes" below